### DISCUSSION TITLE
dist/js/card.js - Failed to open stream: No such file or directory

### DISCUSSION THREAD

**Discussion by haztakki** [laravel/nova-issues#4422] [Q&A] [answered]
We recently started using Flare and noticed that we get this error every time we login. We don't have any cards setup just yet, only setup some fields, actions and some basic metrics. I don't get this error in Ray locally. The issue is non-breaking as far as we can see. Flare just keeps reporting it.

<img width="783" alt="Screenshot 2022-06-14 at 20 10 11" src="https://user-images.githubusercontent.com/20573863/173639998-8b326aa9-ecf6-45f8-9fd9-6ee5ea3676ec.png">

**Here are the metrics we currently have setup (from local dev), nothing special.**

<img width="1131" alt="Screenshot 2022-06-14 at 20 28 21" src="https://user-images.githubusercontent.com/20573863/173640127-2387e916-a279-4e28-9a19-464fb76fd4f3.png">

```<?php

declare(strict_types=1);

namespace App\Nova\Metrics;

use ALCreations\User\Models\Order;
use Laravel\Nova\Http\Requests\NovaRequest;
use Laravel\Nova\Metrics\Value;

class Orders extends Value
{
    /**
     * Calculate the value of the metric.
     *
     * @param  NovaRequest  $request
     * @return mixed
     */
    public function calculate(NovaRequest $request): mixed
    {
        return $this->count($request, model: Order::class);
    }

    /**
     * Get the ranges available for the metric.
     *
     * @return array
     */
    public function ranges(): array
    {
        return [
            30      => __('30 Days'),
            60      => __('60 Days'),
            365     => __('365 Days'),
            'TODAY' => __('Today'),
            'MTD'   => __('Month To Date'),
            'QTD'   => __('Quarter To Date'),
            'YTD'   => __('Year To Date'),
        ];
    }

    /**
     * Get the URI key for the metric.
     *
     * @return string
     */
    public function uriKey(): string
    {
        return 'orders';
    }
}
```

The other two metrics follow the same format, just with a different name. I'm not sure if it is related to the metrics, but I see a metric being as some kind of metric card. I've not had time to dive into this, the issue just kept popping up and I tweeted. I'll spend some time over the weekend to have a look.

**Comment by davidhemphill** [comment, marked as answer]
It looks like you're trying to load a custom tool from the "nova-components" directory. This indicates there's a tool calling `Nova::script` in your codebase.

**Reply by haztakki** [reply]
Yeah, we aren't doing that though. I'll search the code soon for Nova::script but I'm more than certain I won't find anything custom.

**Comment by haztakki** [comment]
OK, so I checked. You was right. I guess at some point I started reading the docs about how to setup cards, I obviously got distracted and completely forgot about it. I'm removed this. Should do the trick! Thanks @davidhemphill! Much appreciated.
```
        Nova::serving(function (ServingNova $event) {
            Nova::script('analytics', __DIR__.'/../dist/js/card.js');
            Nova::style('analytics', __DIR__.'/../dist/css/card.css');
        });
```
